README.md

```diff
@@ -109,6 +110,7 @@ Global parameters apply to all sub-charts and make it easier to configure resour
 | `global.githubEnterprise.isPrivateMode`         | Status of private mode on githubEnterprise instance                                                          | `nil`           |
 | `global.githubEnterprise.disableSSL`            | Disable certificate validation on interaction with githubEnterprise instance                                 | `nil`           |
 | `global.githubEnterprise.app.id`                | App id used in Codacy to integrate with GitHub Apps in GitHub Enterprise                                     | `nil`           |
+| `global.githubEnterprise.app.name`              | The name of the GitHub app to integrate with. Only required if you use the GitHub integration .              | `codacy`        |
```

Comment:
`Only required if you use the GitHub integration` -> `Only required if you use the GitHub Enterprise integration`